ui/yarn.lock

```diff
@@ -11731,7 +11731,7 @@ querystring-es3@^0.2.0, querystring-es3@~0.2.0:
   version "0.2.1"
   resolved "https://registry.yarnpkg.com/querystring-es3/-/querystring-es3-0.2.1.tgz#9ec61f79049875707d69414596fd907a4d711e73"
 
-querystring@0.2.0:
+querystring@0.2.0, querystring@^0.2.0:
```

Comment:
You might need to repush this yarn.lock file. I suggest doing a new `yarn install` and see if it changes. I don't think `querystring@^0.2.0` should be there after the `^` was removed from the package.json file.